src/mlpack/core/tree/binary_space_tree/binary_space_tree_impl.hpp

```diff
@@ -877,6 +879,37 @@ template<typename MetricType,
          template<typename BoundMetricType, typename...> class BoundType,
          template<typename SplitBoundType, typename SplitMatType>
              class SplitType>
+size_t BinarySpaceTree<MetricType, StatisticType, MatType, BoundType,
+    SplitType>::PerformSplit(MatType& data,
+    const size_t begin, const size_t count,
+    const typename UBTreeSplit<BoundType<MetricType>, MatType>::SplitInfo& splitInfo)
```

Comment:
Ah, can you split each parameter onto its own line here?  That may necessitate putting `MatType& data` on its own line too so it lines up with the other parameters.  I know it is a minor comment but it is for consistency. :)